test/doc/core/regular/bit_reverse.cpp

```diff
@@ -3,25 +3,31 @@
 #include <bit>
 
 
-using wide_it = eve::wide<std::uint64_t, eve::fixed<4>>;
+using wide_it = eve::wide<std::uint16_t, eve::fixed<4>>;
 
 int main()
 {
   wide_it pi = {-14, 1, 3, 0};
 
   std::cout << "---- simd" << '\n'
-            << "<- pi              = " << pi << '\n'
-            << "-> bit_reverse(pi)  = " << eve::bit_reverse(pi) << '\n'
-            << "-> sbit_reverse(pi) = " << eve::detail::map(eve::bit_reverse, pi)<< '\n'
+            << "<- pi                         = " << pi << '\n'
+            << "-> bit_reverse(pi)            = " << eve::bit_reverse(pi) << '\n'
+            << "-> sbit_reverse(pi)           = " << eve::detail::map(eve::bit_reverse, pi)<< '\n'
```

Comment:
`eve::detail::map(eve::bit_reverse, pi)` should not be here

Reply:
done